application-forum-ui/src/main/resources/ForumCode/NewFlagSheet.xml

```diff
@@ -60,26 +60,46 @@
   ## sending email
   #set($emailRegex = "[-0-9a-zA-Z.+_]+@[-0-9a-zA-Z.+_]+\.[a-zA-Z]{2,4}")
   #set($adminEmail = $xwiki.getXWikiPreference('admin_email'))
+  #if ("${adminEmail}" == '')
+    ## Get admin email from global administration settings
+    #set ($configDocName = 'Mail.MailConfig')
+    #set ($configDoc = $xwiki.getDocument($configDocName))
+    #set ($configClassName = 'Mail.SendMailConfigClass')
+    #set ($configObj = $configDoc.getObject($configClassName))
+    #set ($adminEmail = $configObj.from)
+  #end
   #if (!$adminEmail.matches($emailRegex))
     #set($adminEmail = '')
   #end
-  #set($moderator = $flagDoc.getValue('moderator'))
-  #if("$!{moderator}" != '')
-    #set($moderatorEmail = $xwiki.getUserName($moderator, "$email", false))
-    #if(!$moderatorEmail.matches($emailRegex))
-      #set($moderatorEmail = '')
+
+  #set ($moderator = $flagDoc.getValue('moderator'))
+  #if ("$!{moderator}" != '')
```

Comment:
In this case, the document might not have an object with a `moderator` property and you can test with `"$!moderator"`. Again, no need of `{}` here.